libc/newhdrgen/yaml/math.yaml

```diff
@@ -192,13 +199,53 @@ functions:
     return_type: float
     arguments:
       - type: float
+  - name: daddf128
+    standards:
+      - llvm_libc_ext
+    return_type: double
+    arguments:
+      - type: float128
+      - type: float128
+    guard: LIBC_TYPES_HAS_FLOAT128
+  - name: daddl
+    standards:
+      - stdc
+    return_type: double
+    arguments:
+      - type: long double
+      - type: long double
+  - name: ddivf128
+    standards:
+      - llvm_libc_ext
+    return_type: double
+    arguments:
+      - type: float128
+      - type: float128
+    guard: LIBC_TYPES_HAS_FLOAT128
```

Comment:
but some of these still aren't added yet.

Reply:
These are some of the basic math functions I completed but did not update the header files. I'd assume these should be added to the corresponding yaml file as well.